frontend/src/components/Preview/presenter.jsx

```
@@ -0,0 +1,42 @@
+            <NonIdealState
+                icon="manual"
+                title="Preview unavailable"
+                description="please download artifact"
+            />
+        </div>
+    }
+
+    if (selectedDocument && selectedDocument.documentText) {
+        const {documentText} = selectedDocument
+        if (isHtml(selectedDocument.displayName)) {
+            return <div dangerouslySetInnerHTML={{__html: documentText}}/>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)